_topic_maps/_topic_map.yml

```diff
@@ -3029,6 +3029,8 @@ Topics:
       File: virt-preparing-cdi-scratch-space
     - Name: Re-using statically provisioned persistent volumes
       File: virt-reusing-statically-provisioned-persistent-volumes
+    - Name: Virtual machine disk enlargement
+      File: virt-vm-disk-enlargment
```

Comment:
We're already discussing this on Slack, but I'm wondering if this should be named something like "Resizing a virtual machine disk" or if you can really only enlarge/expand one (in which case we should mention that). 

Also just noting that "enlargment" is a misspelling (in the file name, here, and in the context statement for the assembly) of "enlargement." It's not a huge deal but we might as well fix it now :)